.github/renovate.json

```diff
@@ -12,11 +14,13 @@
     },
     {
       "matchUpdateTypes": ["minor", "patch"],
-      "matchCurrentVersion": "!/^0/",
-      "automerge": true
+      "matchCurrentVersion": "!/^[~^]?0/",
```

Comment:
This is to avoid `0.`, `~0.` & `^0.` versions, right?

Reply:
Yes. Unless the behavior has changed since last I tested it, `/^0/` will only match `"package-name": "0.x"` and not `"package-name": "~0.x"` or `"package-name": "^0.x"`.

Reply:
Right now we are having PRs like this: https://github.com/IBM/pwa-lit-template/pull/442

Reply:
We would probably want to configure this so that: only auto-merge the `minor` & `patch` from "stable" packages. So no `0.` packages. For the `0.` we prefer confirm it manually in a PR.

Reply:
Okay, then this will do what you want. That PR happened because `~0.1.16` didn't match `!/^0/`, but it will match

`!/^[~^]?0/`.

Reply:
Although you should note that refreshing the lock file with `~0.1.16` will install `0.1.17`. If you don't want any updates at all to `0.x` packages, you should remove the `~` from the version in `package.json`.

Reply:
Thanks again! And you're right! I was wrong; I wanted to say that:
- Only auto-merge the `minor` & `patch` from "stable" packages. Also the `0.` patches.
- And for the `0.` minors we prefer confirm it manually in a PR.

Reply:
Okay, I've split it up so 0.x minors will get a PR. 0.x patches and stable minors/patches will get auto-merged.